src/components/charts/shared/observationsTables/ObservationsTable.jsx

```diff
@@ -7,27 +7,26 @@ import { tableTitle } from './observations-tables.module.scss';
 
 class ObservationsTable extends React.PureComponent {
   getCell(answers, cell) {
-    const { accessor, id, ids, data, content } = cell;
+    const { accessor, id, data, content, type } = cell;
```

Comment:
`type` is added to format display for eccentricity/inclination using the existing formatting helper.